demisto_sdk/commands/content_graph/strict_objects/pack_meta_data.py

```diff
@@ -0,0 +1,65 @@
+from typing import List, Optional, Union
+
+from pydantic import Field
+
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+from demisto_sdk.commands.content_graph.strict_objects.common import BaseStrictModel
+
+
+class StrictPackMetadata(BaseStrictModel):
+    name: str
+    display_name: Optional[str] = None
+    description: Optional[str] = None
+    created: Optional[str] = None
+    updated: Optional[str] = None
+    legacy: Optional[bool] = None
+    support: str
+    url: Optional[str] = None
+    email: Optional[str] = None
+    eula_link: Optional[str] = Field(None, alias="eulaLink")
+    author: str
+    author_image: str = Field("", alias="authorImage")
+    certification: Optional[str] = None
+    price: Optional[int] = None
+    hidden: Optional[bool] = None
+    server_min_version: Optional[str] = Field(alias="serverMinVersion")
+    current_version: Optional[str] = Field(alias="currentVersion")
```

Comment:
can we use `packaging.Version` to validate the value here?

Reply:
Sure, adding a validator